### PR TITLE
Don't propagate noret to fcn if fcn jmps outside itself

### PR DESCRIPTION
**Pull request by kazarmy**
https://github.com/radareorg/radare2-regressions/commit/62df5a9ec756d146d21ece9ec274a7a5cdb6f961
Fixes https://github.com/radareorg/radare2/pull/15141#issuecomment-536556946.

### CODE REVIEW

File: libr/core/canal.c [hunk 1]
```diff
@@ -5495,6 +5495,13 @@ static bool is_noreturn_function(RCore *core, RAnalFunction *f) {
 			case R_ANAL_OP_TYPE_RET:
 				r_anal_op_free (op);
 				return false;
+			case R_ANAL_OP_TYPE_JMP: {
+				RAnalFunction *target_f = r_anal_get_fcn_in (core->anal, op->jump, 0);
+				if (!target_f || target_f->addr != f->addr) {
+					r_anal_op_free (op);
+					return false;
+				}
```
Comment (radare) [inline libr/core/canal.c]: uhm static analysis tools will fail here because of the missing break

**Comment by radare** [comment]
looks red to me

**Comment by radare** [comment]
ping

**Comment by kazarmy** [comment]
Superseded by #15190.

File: libr/core/canal.c (hunk 1 above)
Comment (kazarmy) [reply libr/core/canal.c]: ok 5e8c6912fae61fb4072578f701b9ae5961e6335d

**Pull request closed without merging**